src/features/wallet/root/ActivityDetails/PaymentItem.tsx

```diff
@@ -0,0 +1,87 @@
+import React from 'react'
+import { useTranslation } from 'react-i18next'
+import Box from '../../../../components/Box'
+import Text from '../../../../components/Text'
+import MyAccount from '../../../../assets/images/myAccount.svg'
+import TheirAccount from '../../../../assets/images/theirAccount.svg'
+
+type Props = {
+  text: string
+  subText?: string | null
+  title?: string | null
+  isFirst?: boolean
+  isLast?: boolean
+  mode: 'to' | 'from' | 'memo' | 'location' | 'seller' | 'owner' | 'buyer'
+  isMyAccount?: boolean
+}
+const PaymentItem = ({
+  text,
+  subText,
+  isFirst = true,
+  isLast = false,
+  mode,
+  title,
+  isMyAccount,
+}: Props) => {
+  const { t } = useTranslation()
+
+  const showIcon =
+    mode === 'to' || mode === 'from' || mode === 'seller' || mode === 'buyer'
+
+  return (
+    <Box
+      height={63}
+      backgroundColor="grayBox"
+      alignItems="center"
+      padding="ms"
+      flexDirection="row"
+      borderTopLeftRadius={isFirst ? 'm' : 'none'}
+      borderTopRightRadius={isFirst ? 'm' : 'none'}
+      borderBottomLeftRadius={isLast ? 'm' : 'none'}
+      borderBottomRightRadius={isLast ? 'm' : 'none'}
+      justifyContent="space-between"
+      marginBottom={isLast ? 'l' : 'xxxs'}
+    >
+      <Text
+        variant="medium"
+        fontSize={15}
+        color="black"
+        flex={1}
+        alignSelf="flex-start"
+      >
+        {title || t(`activity_details.${mode}`)}
+      </Text>
+      <Box
+        alignItems="flex-end"
+        justifyContent="space-between"
+        flex={1}
+        marginHorizontal="ms"
+      >
+        <Text
+          variant="medium"
+          fontSize={14}
+          color="black"
+          numberOfLines={1}
+          ellipsizeMode="middle"
+        >
+          {text}
+        </Text>
+        {(isMyAccount || subText) && (
+          <Text
+            variant="light"
+            fontSize={12}
+            color="black"
+            marginLeft="s"
+            marginTop="xs"
+          >
+            {subText || t('activity_details.my_account')}
+          </Text>
+        )}
+      </Box>
+      {isMyAccount && showIcon && <MyAccount />}
```

Comment:
I think we can leave this images out for now. They are normally generate by https://github.com/allenan/react-identicon-variety-pack which im not sure is compatible with react native or not but I think it might be using Canvas

Reply:
![IMG_30F8346603AA-1](https://user-images.githubusercontent.com/1760821/106001806-0585e300-6076-11eb-8e55-d8d10561c342.jpeg)
@allenan Is that right, should I leave this icons out?